From 5410734d41de1e6d9af6dfc7eee05c867e241458 Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <lukma@denx.de>
Date: Wed, 3 Mar 2021 10:52:30 +0100
Subject: [PATCH 1/5] dts: Remove fec1 related description

fec1 is not used in the new HW setup (with mv88e6020) for the inverter
board.

Signed-off-by: Lukasz Majewski <lukma@denx.de>
---
 arch/arm/boot/dts/imx6ull-kie-inverter.dts | 29 ----------------------
 1 file changed, 29 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-kie-inverter.dts b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
index f17669c86caa..d01279ba329a 100644
--- a/arch/arm/boot/dts/imx6ull-kie-inverter.dts
+++ b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
@@ -17,7 +17,6 @@
 		mmc0 = &usdhc2;
 		usb0 = &usbotg1;
 		ethernet0 = &fec2;
-		ethernet1 = &fec1;
 	};
 
 	backlight_display: backlight-display {
@@ -147,19 +146,6 @@
 	status = "okay";
 };
 
-&fec1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_enet1>;
-	phy-mode = "rmii";
-	phy-handle = <&ethphy0>;
-	//phy-reset-gpios = <&gpio3 18 GPIO_ACTIVE_LOW>;
-	//phy-reset-duration = <1>;
-	//phy-reset-post-delay = <1>;
-	//phy-supply = <&reg_3v3>;
-	status = "okay";
-};
-
-
 &fec2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet2>;
@@ -254,21 +240,6 @@
 &iomuxc {
 	pinctrl-names = "default";
 
-	pinctrl_enet1: enet1grp {
-		fsl,pins = <
-			MX6UL_PAD_ENET1_RX_EN__ENET1_RX_EN	0x1b0b0
-			MX6UL_PAD_ENET1_RX_ER__ENET1_RX_ER	0x1b0b0
-			MX6UL_PAD_ENET1_RX_DATA0__ENET1_RDATA00	0x1b0b0
-			MX6UL_PAD_ENET1_RX_DATA1__ENET1_RDATA01	0x1b0b0
-			MX6UL_PAD_ENET1_TX_EN__ENET1_TX_EN	0x1b010
-			MX6UL_PAD_ENET1_TX_DATA0__ENET1_TDATA00	0x1b010
-			MX6UL_PAD_ENET1_TX_DATA1__ENET1_TDATA01	0x1b010
-			MX6UL_PAD_ENET1_TX_CLK__ENET1_REF_CLK1	0x4001b031
-			MX6UL_PAD_LCD_DATA17__GPIO3_IO22	0x17059
-			MX6UL_PAD_LCD_DATA18__GPIO3_IO23	0x17059
-		>;
-	};
-
 	pinctrl_enet2: enet2grp {
 		fsl,pins = <
 			MX6UL_PAD_GPIO1_IO06__ENET2_MDIO	0x10010
-- 
2.20.1

